interpreter/cling/include/cling/Utils/ASTImportSource.h

```diff
@@ -0,0 +1,88 @@
+#ifndef CLING_ASTIMPORTSOURCE_H
+#define CLING_ASTIMPORTSOURCE_H
+
+#include "cling/Interpreter/Interpreter.h"
+
+#include "clang/Frontend/CompilerInstance.h"
+#include "clang/Sema/Sema.h"
+#include "clang/AST/ASTContext.h"
+#include "clang/AST/ASTImporter.h"
+
+#include <string>
+#include <map>
+
+namespace clang {
+  class ASTContext;
+  class Expr;
+  class Decl;
+  class DeclContext;
+  class DeclarationName;
+  class GlobalDecl;
+  class FunctionDecl;
+  class IntegerLiteral;
+  class NamedDecl;
+  class NamespaceDecl;
+  class NestedNameSpecifier;
+  class QualType;
+  class Sema;
+  class TagDecl;
+  class TemplateDecl;
+  class Type;
+  class TypedefNameDecl;
+}
+
+namespace cling {
+  namespace utils {
+
+    class ASTImportSource : public clang::ExternalASTSource {
```

Comment:
Can you mark all virtual functions that your class re-implements with `override`?